back-end/helpers/transcript_data_formatter.spec.js

```diff
@@ -114,7 +114,7 @@ describe("formatTranscript", () => {
         const received = await TranscriptFormatter.formatTranscript(globalDatabase.rawTranscript);
         expect(received.length).toBe(3);
         expect(received[0].children).toEqual(children);
-        expect(received[0].totalChildren).toBe(2);
+        expect(received[0].total_children).toBe(2);
```

Comment:
Nice catches on these. I think we should reconfirm with the team tmrw on discord on how we want to name things, because I think we have a little mix of some variables being camel case and some using underscores and I want to make sure we're all on the same page from here on out so that we get less of these mix-ups.